frontend/js/player_adapter_HTML5.js

```diff
@@ -77,98 +80,105 @@ define(["jquery",
 
                 targetElement.style.width = "100%";
                 targetElement.style.height = "100%";
+                targetElement.preload = "auto";
 
+                window.hls = hls;
```

Comment:
Note that the code does not work anymore.

My remark about changing `Hls` to `hls` only referred to the RequireJS module identifier. The name of the exported object can be `Hls` if we want, and the name of the global object you assign to here **has** to be `Hls`, because that is apparently what MediaElement.js is looking for.

Reply:
Thanks for notice that!

I was deceived, in safari works, now I tried with Chrome and makes that issue